dan_lombardy/test/messages_tests.js

```diff
@@ -0,0 +1,69 @@
+"use strict";
+
+var chai = require('chai');
+var chaihttp = require('chai-http');
+chai.use(chaihttp);
+var expect = chai.expect;
+var mongoose = require('mongoose');
+var Message = require(__dirname + '/../models/message');
+
+process.env.MONGOLAB_URI = 'mongodb://localhost/message_test';
+require(__dirname + '/../server');
+
+describe('message routes post', function(){
+  after(function(done){
+    mongoose.connection.db.dropDatabase(function(){
+      done();
+    });
+  });
+
+  it('should post a message', function(done){
+    var messageData = {"oneWordTitle": "testMsg", "secretToRead":"Run away!", "priority":5, "views":0, "destructMessage":"default message"};
+    chai.request('localhost:3000')
+    .post('/api/messages')
+    .send(messageData)
+    .end(function(err, res){
+      expect(err).to.eql(null);
+      expect(res.body).eql({msg: 'It posted successfully!'});
+      done();
+    });
+  });
+});
+
+describe('message routes gets', function(){
+  after(function(done){
+    mongoose.connection.db.dropDatabase(function(){
+      done();
+    });
+  });
+
+    before(function(done){
+      (new Message({"oneWordTitle": "testMsg", "secretToRead":"Run away!", "priority":5, "views":0, "destructMessage":"default message"}))
+      .save(function(err, data){
+       expect(err).to.eql(null);
+       this.message = data;
+       done();
+    }.bind(this));
+  });
+
+    it('should get a single message', function(done){
+      chai.request('localhost:3000')
+      .get('/api/messages/' + 'testMsg')
+      .end(function(err, res){
+        expect(err).to.eql(null);
+        expect(res.body.oneWordTitle).to.equal('testMsg');
+        done();
+      });
+    });
+
+    it('should get all the messages', function(done){
+      chai.request('localhost:3000')
+      .get('/api/messages')
+      .end(function(err, res){
+        expect(err).to.eql(null);
+        expect(res.body).to.eql([ "testMsg" ]);
+        done();
+      });
+    });
+
```

Comment:
You should have a test that checks a message is deleted when the right conditions are met.